cmd/shipper-state-metrics/math.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"errors"
+	"math"
+	"sort"
+)
+
+// Lifted from https://github.com/montanaflynn/stats
```

Comment:
This should probably be mentioned in NOTICE as well? I don't really know but worth checking.